connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TaskConfig.java

```diff
@@ -49,6 +49,6 @@ public TaskConfig() {
     }
 
     public TaskConfig(Map<String, ?> props) {
-        super(config, props);
+        super(config, props,true);
```

Comment:
Little late, but I fixed this. As you mentioned the missing space cause the build to fail for checkstyle reasons but the rest of the tests pass. Should we go ahead and reopen this to add the space in?